packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -268,6 +264,28 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
             )}
           </DrawerActions>
         </DrawerHead>
+        <div className="pf-chatbot__input">
```

Comment:
I'd rename this "pf-chatbot__heading-container" or "title-container" or something like that. You'll also want to update the CSS file. We want to leave the existing className as-is on the input div, ideally, in case someone has tied any custom overrides to this class.